docs/02-app/01-building-your-application/06-optimizing/10-open-telemetry.mdx

```diff
@@ -27,10 +27,10 @@ That's why we prepared a package `@vercel/otel` that helps you get started quick
 
 ### Using `@vercel/otel`
 
-To get started, you must install `@vercel/otel`:
+To get started, you must install `@vercel/otel`, `@opentelemetry/api`, and `@opentelemetry/api-logs`:
```

Comment:
Do we really need `@opentelemetry/api-logs`?

Reply:
Well, it didn't seem to work without on my end, but please test to see if that is correct! 😄